Composer/packages/client/src/recoilModel/dispatchers/utils/project.ts

```diff
@@ -394,7 +394,7 @@ export const initBotState = async (callbackHelpers: CallbackInterface, data: any
   }
   set(schemasState(projectId), schemas);
   set(localeState(projectId), locale);
-  set(botDiagnosticsState(projectId), [...diagnostics, ...botFiles.diagnostics]);
+  set(botDiagnosticsState(projectId), diagnostics);
```

Comment:
```, ...botFiles.diagnostics```
projectTree will use those diagnostics to show error & warning.

Reply:
![image](https://user-images.githubusercontent.com/39758135/99365430-3d441200-28f2-11eb-8bf2-ed1e4421c92a.png)
I found the diagnostics in projecttree is from the botProjectSpaceSelector